src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void machineWork() {
+        Machine[] machines = new Machine[]{new Excavator(),new Bulldozer(),new Truck()};
```

Comment:
```suggestion
        Machine[] machines = new Machine[]{new Excavator(), new Bulldozer(), new Truck()};
```
Let’s use white spaces ;)